### PR TITLE
fix: nav bar height mismatch

### PR DESCRIPTION
**Pull request by zchsh**
## 🔗 Relevant links

- [Preview link][preview] 🔎
- [Asana task][task] 🎟️

## 🗒️ What

Fixes minor issues in the top nav bar that caused it to have an unexpected height on certain viewport sizes and on certain pages.

## 🤷 Why

To fix issues related to layouts that rely on the sticky top nav bar being a known, consistent height.

## 🛠️ How

- adjusts top nav padding so that height is accurate
- adjusts home page logo padding so nav height is consistent between product-specific and home pages

## 📸 Design Screenshots

Nav bar

## 🧪 Testing

- [ ] Visit the [home page][preview], and ensure the top nav-bar height is consistently `68px` across all viewports
- [ ] Visit a [product page][/waypoint], and ensure the top nav-bar height is consistently `68px` across all viewports

[task]: https://app.asana.com/0/1202114367927919/1202316169578550/f
[preview]: https://dev-portal-git-zsfix-top-bar-height-hashicorp.vercel.app
[/waypoint]: https://dev-portal-git-zsfix-top-bar-height-hashicorp.vercel.app/waypoint

### CODE REVIEW

File: src/components/navigation-header/navigation-header.module.css [hunk 1]
```diff
@@ -22,8 +22,8 @@
   width: 100%;
 
   @media (--dev-dot-tablet-down) {
-    padding-bottom: 16px;
-    padding-top: 16px;
+    padding-bottom: 14px;
+    padding-top: 14px;
```
Comment (zchsh) [inline src/components/navigation-header/navigation-header.module.css:26]: This is intended to fix the issue on all product logo pages. Previously nav bar height on mobile was `72px`, it should now be `68px`. Thanks Ashlee for tracking this down! 🙌

File: src/components/navigation-header/components/home-page-content/home-page-content.module.css [hunk 2]
```diff
@@ -1,3 +1,13 @@
 .siteLogo {
   display: flex;
+
+  /* Note: padding here necessary to match height of .productLogo
+     in navigation-header.module.css */
+  padding-top: 4px;
+  padding-bottom: 4px;
+
+  & svg {
+    height: 32px;
```
Comment (zchsh) [inline src/components/navigation-header/components/home-page-content/home-page-content.module.css:10]: This is intended to fix height inconsistency issues on the home page, where the logo takes on a different height (per design specs).

**Comment by zchsh** [comment]
@ashleemboyer This PR fixes the top nav height inconsistency, as discussed in Slack ☺️

**Comment by zchsh** [comment]
@ashleemboyer I think this fixes the immediate issue, at least when text size isn't manually adjusted, so I had not planned on diving into the `rem` vs `px` issue in the top nav until later.

Either way, would definitely want to handle that in a separate PR! Aim here is to unblock the sidebar mobile refinement ☺️

**Review by ashleemboyer — APPROVED** [review]

**Pull request merged**